catppuccin_options_tmux.conf

```diff
@@ -37,6 +37,16 @@ set -ogq @catppuccin_window_current_color "#{@thm_surface_1}"
 set -ogq @catppuccin_window_current_background "#{@thm_peach}"
 set -ogq @catppuccin_window_current_text " ##T"
 set -ogq @catppuccin_window_number_position "left"
+set -ogq @catppuccin_window_status "icon"
+set -ogq @catppuccin_window_flags_icon_last " 󰖰"
+set -ogq @catppuccin_window_flags_icon_current " 󰖯"
+set -ogq @catppuccin_window_flags_icon_zoom " 󰁌"
+set -ogq @catppuccin_window_flags_icon_mark " 󰃀"
+set -ogq @catppuccin_window_flags_icon_silent " 󰂛"
+set -ogq @catppuccin_window_flags_icon_activity " 󱅫"
+set -ogq @catppuccin_window_flags_icon_bell " 󰂞"
+# icon order: #!~[*-]MZ
```

Comment:
Can you expand on this comment a bit? It looks useful, but needs a bit more commentary

Reply:
Updated the comments, hope it's clear now.
TLDR that's the icon order tmux uses/displays when using text flags (`#F`)